pdns/rpzloader.cc

```diff
@@ -363,7 +363,7 @@ void RPZIXFRTracker(const std::vector<ComboAddress>& masters, boost::optional<DN
     return;
   }
 
-  time_t refresh;
+  uint32_t refresh = std::max(refreshFromConf ? refreshFromConf : oldZone->getRefresh(), 10U);
```

Comment:
Perhaps add a comment explaining that if the zone has not been loaded yet, the refresh will default to 0 so we will pick 10?